<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0" stdsetdef="1">
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>KCurrencyEditDlgDecl</class>
  <widget class="QDialog" name="KCurrencyEditDlgDecl">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>666</width>
        <height>516</height>
      </rect>
    </property>
    <property name="windowTitle">
      <string>Currencies</string>
    </property>
    <property name="sizeGripEnabled">
      <bool>true</bool>
    </property>
    <layout class="QVBoxLayout">
      <item>
        <layout class="QHBoxLayout">
          <item>
            <layout class="QVBoxLayout">
              <item>
                <widget class="K3ListView" name="m_currencyList"/>
              </item>
              <item>
                <widget class="Q3Frame" name="m_baseCurrencyFrame">
                  <property name="frameShape">
                    <enum>QFrame::NoFrame</enum>
                  </property>
                  <property name="frameShadow">
                    <enum>QFrame::Plain</enum>
                  </property>
                  <layout class="QVBoxLayout">
                    <property name="margin">
                      <number>0</number>
                    </property>
                    <item>
                      <layout class="QHBoxLayout">
                        <item>
                          <spacer name="spacer2">
                            <property name="sizeHint">
                              <size>
                                <width>51</width>
                                <height>21</height>
                              </size>
                            </property>
                            <property name="sizeType">
                              <enum>QSizePolicy::Expanding</enum>
                            </property>
                            <property name="orientation">
                              <enum>Qt::Horizontal</enum>
                            </property>
                          </spacer>
                        </item>
                        <item>
                          <widget class="KPushButton" name="m_baseCurrencyButton">
                            <property name="text">
                              <string>Select as base currency</string>
                            </property>
                          </widget>
                        </item>
                        <item>
                          <spacer name="spacer3">
                            <property name="sizeHint">
                              <size>
                                <width>61</width>
                                <height>31</height>
                              </size>
                            </property>
                            <property name="sizeType">
                              <enum>QSizePolicy::Expanding</enum>
                            </property>
                            <property name="orientation">
                              <enum>Qt::Horizontal</enum>
                            </property>
                          </spacer>
                        </item>
                      </layout>
                    </item>
                  </layout>
                </widget>
              </item>
            </layout>
          </item>
          <item>
            <widget class="Q3GroupBox" name="m_detailGroup">
              <property name="title">
                <string>Details</string>
              </property>
              <layout class="QVBoxLayout">
                <item>
                  <layout class="QGridLayout">
                    <item row="1" column="0">
                      <widget class="QLabel" name="textLabel3">
                        <property name="text">
                          <string>Symbol</string>
                        </property>
                        <property name="wordWrap">
                          <bool>false</bool>
                        </property>
                      </widget>
                    </item>
                    <item row="0" column="1">
                      <widget class="QLabel" name="m_idLabel">
                        <property name="text">
                          <string>real ID</string>
                        </property>
                        <property name="wordWrap">
                          <bool>false</bool>
                        </property>
                      </widget>
                    </item>
                    <item row="1" column="1">
                      <widget class="kMyMoneyLineEdit" name="m_symbolEdit"/>
                    </item>
                    <item row="0" column="0">
                      <widget class="QLabel" name="textLabel1">
                        <property name="text">
                          <string>ID</string>
                        </property>
                        <property name="wordWrap">
                          <bool>false</bool>
                        </property>
                      </widget>
                    </item>
                  </layout>
                </item>
                <item>
                  <widget class="QLabel" name="m_description">
                    <property name="text">
                      <string>X</string>
                    </property>
                    <property name="wordWrap">
                      <bool>false</bool>
                    </property>
                  </widget>
                </item>
                <item>
                  <widget class="K3ListView" name="m_onlineSourceTable">
                    <column>
                      <property name="text">
                        <string>Currency</string>
                      </property>
                      <property name="clickable">
                        <bool>true</bool>
                      </property>
                      <property name="resizable">
                        <bool>true</bool>
                      </property>
                    </column>
                    <column>
                      <property name="text">
                        <string>Online Source</string>
                      </property>
                      <property name="clickable">
                        <bool>true</bool>
                      </property>
                      <property name="resizable">
                        <bool>true</bool>
                      </property>
                    </column>
                  </widget>
                </item>
              </layout>
            </widget>
          </item>
        </layout>
      </item>
      <item>
        <layout class="QHBoxLayout">
          <property name="margin">
            <number>0</number>
          </property>
          <property name="spacing">
            <number>6</number>
          </property>
          <item>
            <widget class="QPushButton" name="buttonHelp">
              <property name="enabled">
                <bool>true</bool>
              </property>
              <property name="text">
                <string>Help</string>
              </property>
              <property name="shortcut">
                <string>F1</string>
              </property>
              <property name="autoDefault">
                <bool>true</bool>
              </property>
            </widget>
          </item>
          <item>
            <spacer name="Horizontal Spacing2">
              <property name="sizeHint">
                <size>
                  <width>20</width>
                  <height>20</height>
                </size>
              </property>
              <property name="sizeType">
                <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="orientation">
                <enum>Qt::Horizontal</enum>
              </property>
            </spacer>
          </item>
          <item>
            <widget class="QPushButton" name="buttonClose">
              <property name="text">
                <string>Close</string>
              </property>
              <property name="autoDefault">
                <bool>true</bool>
              </property>
              <property name="default">
                <bool>true</bool>
              </property>
              <property name="toolTip" stdset="0">
                <string>If this button is not activated, select the base currency first.</string>
              </property>
              <property name="whatsThis" stdset="0">
                <string>Press this button to close the dialog. If the button is not activated, select your base currency.</string>
              </property>
            </widget>
          </item>
        </layout>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="6" margin="11"/>
  <customwidgets>
    <customwidget>
      <class>kMyMoneyLineEdit</class>
      <extends>QWidget</extends>
      <header location="local">../widgets/kmymoneylineedit.h</header>
      <sizehint>
        <width>90</width>
        <height>25</height>
      </sizehint>
      <container>0</container>
      <sizepolicy>
        <hordata>5</hordata>
        <verdata>5</verdata>
      </sizepolicy>
      <pixmap>image0</pixmap>
    </customwidget>
    <customwidget>
      <class>K3ListView</class>
      <extends>Q3ListView</extends>
      <header location="global">k3listview.h</header>
    </customwidget>
  </customwidgets>
  <images>
    <image name="image0">
      <data format="PNG" length="1114">89504e470d0a1a0a0000000d4948445200000016000000160806000000c4b46c3b0000042149444154789c8d95c18b1c4514c67fbb29e5b54ea05b26da1d36b08d096e8b11261871837bc8e282ac7ac82187e049a317e32d37110992bf42123c694eea414c0e0bf1b0ec6c402139485a54e8810c54c10e741146fa110bd643f74c12bd642e4d75f77cf5d5ef7dafdfc2ce7087d9eff32fdfdff74e5114104011e489d6e952cce5cfbe5998692dec0c77b87aedd3fddd9f2b3efce02291082209aa3504685022e4ffd7a000a82a765c515615ce95acae659c3f7765c100ec6e575cf8f822e991025f3b6a6f890c484f181ccfc1583428f5d4e2fe84da039aa04121b40eb32483a0ec6e5bce9f8385d36fa7fb9b1b17c8f31cbfe7a95c45be94520c72b2141a1caa160d1666e24ea94aa11ee7685054159d2ada34d41387c40dc63b8844f0b5c73a4bb19cb3f9de001f2aac1fa23aa209168c004224902e29d22ff1e38abbdb275055c4800610233867311a1491847a62c98f66ac6f14d8e99072f22daa15b68a8108424d6422e24c88e3966fbce4c9d786945b27f0be0103ed13c16004d59a2c4d58df28a87548e9aee2274ab95d60c7193c9a0281fc959ae2a4450364a9e2572c7a2b43438374ef2dd2154012d050514dbec7396578a3130d9d6850066f0ad9d186f2d78cdbbb799b0a202f2c48d3aebbb418310240b112637588574bf54b8efaa4fd9b691dacbf5b305813542bae63a9eee4c4694d967b5494eca8c7ef0933bdc5d90e6a2caa0e3f013b7adce9fa5b03d637561113a30106a71d18c5fd9677058338ad01e91c0b8b743b880182ce19cd9c0e5e2b583fbdda565d3d04cfedad180942133af201b2bc3588e111c680068f489780ce69b192b279768088524f2b3458ea694d3de99a4323bc976ed34eb86b98c51993f6be10f77240c99685cd731991f1582df15aa15a213d583f5313a7806988fb8a0640e336359d9e9933568bf44062214ba19e2a7652a13d8b068ff596b807182189234e9da9f1ceb78c81b28c3ac6fe21e306c58e41d5118952acd6a0c2f56b9e6a5c72f346cdcd6b19a3bf8010010d714f89533b674c90c7189b598eab12a45fb5c75b8253672b863fbcccf5af93b6904119fe9851bf5153ac3a746ae7b9d5698c1dc58f313e70e4707ce9d84bafa2f79fc23fb8c7c1438ef04031a2642fee21060ec69ee47020796ecaa12313ccd333d1b67935047acf2be3bb11aa010dbe656cc755dbe3b772084a7ebc6d5711253b593dfcc0cc8fcd3cafda455512c7f200ec56cbfac0f1d7d34b7fdf0f98fd88a92a7e9cb0e70cc933c0b38a2c823ea0bd76c70c016adb67f47bc2a117a6d40abeea33fa23c2fb867edfb413e4a34fded94ffb3991c46dc7754cd3634ad2af490b857f5a5137126a97e02a8120a485235b6a28ef80bd077ee2b9f2d54fed04595dcbd8ddb6c4bd87334c10cadb2064e856177e7864e669fbe1df154a0142839f7ad2e5b6880bff1da66eece78e315d959f609da670f98befe6c3f45fb42672c9acff7f660000000049454e44ae426082</data>
    </image>
  </images>
</ui>
